crates/eips/src/eip2718.rs

```diff
@@ -124,6 +124,16 @@ pub trait Decodable2718: Sized {
             .unwrap_or_else(|| Self::fallback_decode(buf))
     }
 
+    /// Decode a transaction according to [EIP-2718], ensuring no trailing bytes.
```

Comment:
let's a bit more context here, that this is intended to decode a single transaction from the entire buffer without leaving any trailing bytes